src/Integration.Vsix/Suppression/LiveIssue.cs

```diff
@@ -33,7 +34,9 @@ public class LiveIssue
             string wholeLineText = "")
         {
             Diagnostic = diagnostic;
-            IssueFilePath = issueFilePath;
+            IssueFilePath = issueFilePath != string.Empty 
```

Comment:
I couldn't find cases where the file path was null but maybe I should try to be defensive here, WDYT?

Reply:
Hhhmm. It should be clear to someone reading the code later whether nulls are allowed or not, so there are three options: assert, throw, or be defensive and handle nulls. The simplest is probably to be defensive and use _IsNullOrEmpty_.